src/libs/ajax/SamResources.ts

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash/fp';
+import { appIdentifier, authOpts, fetchSam, jsonBody } from 'src/libs/ajax/ajax-common';
+
+export const SamResources = (signal: AbortSignal) => ({
+  leave: (samResourceType, samResourceId): Promise<void> =>
+    fetchSam(
+      `api/resources/v2/${samResourceType}/${samResourceId}/leave`,
+      _.mergeAll([authOpts(), appIdentifier, { method: 'DELETE' }])
+    ),
+
+  getSignedUrl: async (bucket: string, object: string, project: string, noRequesterPays = false): Promise<string> => {
```

Comment:
nit - rename `noRequesterPays` to `isRequesterPays` or `shouldRequesterPay` or something along those lines.  Key thing would be inverting the boolean so that insted of `noFoo` it becomes `foo`

Reply:
ehh, I actually agree. Lemme get a Sam PR up

Reply:
https://github.com/broadinstitute/sam/pull/1104